api/core/v2/check.proto

```diff
@@ -32,7 +32,8 @@ message CheckRequest {
     int64 Issued = 4 [(gogoproto.jsontag) = "issued"];
 
     // HookAssets is a map of assets required to execute hooks.
-    map<string, AssetList> hook_assets = 5 [(gogoproto.jsontag) = "hook_assets"];
```

Comment:
🤔did vim reformat hook assets and assets?

Reply:
Yeah, it's just the auto-formatter.